packages/nc-gui/components/tabs/auth/ApiTokenManagement.vue

```diff
@@ -106,17 +106,29 @@ onMounted(() => {
         </template>
       </a-button>
       <!-- Generate Token -->
-      <div class="flex flex-row justify-center w-full -mt-1">
+      <div class="flex flex-row justify-center w-full -mt-1 mb-3">
         <a-typography-title :level="5">{{ $t('title.generateToken') }}</a-typography-title>
       </div>
       <!-- Description -->
-      <div class="flex flex-col mt-3 justify-center space-y-6">
+      <a-form
+        ref="form"
+        :model="selectedTokenData"
+        name="basic"
+        layout="vertical"
+        class="flex flex-col justify-center space-y-6"
+        no-style
+        autocomplete="off"
+        @finish="generateToken"
+      >
         <a-input v-model:value="selectedTokenData.description" :placeholder="$t('labels.description')" />
+
         <!-- Generate -->
         <div class="flex flex-row justify-center">
-          <a-button type="primary" @click="generateToken"> {{ $t('general.generate') }} </a-button>
+          <button type="submit" class="ant-btn ant-btn-primary">
```

Comment:
Why use `button` and apply `ant-btn` and not just `a-button`? 😄 
if you need to add `type="submit"` you can pass it as `html-type` prop, i.e. `html-type="submit"`

```html
<a-button html-type="submit" type="primary">
   {{ $t('general.generate') }}
</a-button>
```

Reply:
That's a dummy mistake lol. I'll do the fix.

Reply:
@bcakmakoglu done!